docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -2,69 +2,135 @@
 [[ml-configuring-aggregation]]
 = Aggregating data for faster performance
 
-By default, {dfeeds} fetch data from {es} using search and scroll requests.
-It can be significantly more efficient, however, to aggregate data in {es}
-and to configure your {anomaly-jobs} to analyze aggregated data.
+Aggregating data in {es} and configuring the {anomaly-jobs} to analyze this
+aggregated data is more efficient than the default behavior when {dfeeds} fetch 
+data from {es} using search and scroll requests. 
 
-One of the benefits of aggregating data this way is that {es} automatically
-distributes these calculations across your cluster. You can then feed this
-aggregated data into the {ml-features} instead of raw results, which
-reduces the volume of data that must be considered while detecting anomalies.
+When you aggregate data, {es} automatically distributes the calculations across 
+your cluster. Then you can feed this aggregated data into the {ml-features} 
+instead of raw results. It reduces the volume of data that must be analyzed.
+
+
+[discrete]
+[[aggs-requs-dfeeds]]
+== Requirements
+
+There are a number of requirements for using aggregations in {dfeeds}.
```

Comment:
I think I would address this question in a separate PR. It would give me time to think about how to display this information and collect that information.